R/plot.R

```diff
@@ -494,7 +494,7 @@ setMethod(
     "plot",
     signature(x = "SimInf_model", y = "ANY"),
     function(x, y, level = 1, index = NULL, range = 0.5, type = "s",
-             lwd = 2, frame.plot = FALSE, legend = TRUE, ...) {
+             lwd = 2, frame.plot = FALSE, legend = TRUE, log_transform = FALSE, ...) { #ajab
```

Comment:
I suggest that we change the parameter name from `log_transform=FALSE` to `log=""` for consistency with the R plot function https://search.r-project.org/R/refmans/graphics/html/plot.default.html

This would also enable log-values of the x-axis.